tcf_website/management/commands/upload_csv_review_drive.py

```diff
@@ -0,0 +1,174 @@
+    # Visually check that CS department has been moved to E School and that
+    # other schools have been split up.
+
+    help = "Uploads CSV from /review_drive_responses/ in the format of Time, Email, Mnemonic, Number, Name, Review, " \
+           "Rating, Enjoyable, Recommendation, Difficulty, Reading Hours, Writing Hours, Group work Hours, Misc Hours"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "filename",
+            help=(
+                'Filepath of csv to upload in /review_drive_responses/'
+                "Must be in correct format."
+            ),
+            type=str,
+        )
+        parser.add_argument(
+            "season",
+            help=(
+                'What season the reviews took place i.e. Fall, January, Spring, Summer'
+            ),
+            type=str,
+        )
+        parser.add_argument(
+            "year",
+            help=(
+                'Year of reviews i.e. 2024'
+            ),
+            type=int,
+        )
+
+        # Named (optional) arguments
+        parser.add_argument(
+            "--verbose",
+            action="store_true",
+            help="Verbose output",
+        )
+
+    def handle(self, *args, **options):
+        verbose = options["verbose"]
+        filename = options["filename"]
+        season = options["season"]
+        year = options["year"]
+
+        # Get semester object from args
+        semester_query = Semester.objects.filter(season=season.upper(), year=year)
+        if semester_query.count() <= 0:
+            print('ERROR: Semester not found')
+            return
+        semester = semester_query.first()
+
+        # Needs to be created ahead of time in the db
+        dummy_account = User.objects.filter(computing_id__iexact='reviewdrive')
+        if dummy_account.count() <= 0:
+            print('ERROR: DUMMY ACCOUNT NOT FOUND')
+            return
+
+        if verbose:
+            print('Starting file upload...')
+
+        file = open(os.path.join(DATA_DIR, filename), mode='r')
+        csv_file = csv.reader(file)
+
+        for i, line in enumerate(csv_file):
+            if i == 0:
+                continue
+
+            # Strip needed to remove random whitespace for exact match
+            email = line[1].strip()
+            mnemonic = line[2].strip()
+            num = int(line[3].strip())
+            instructor = line[4].strip().split(' ', 1)
+
+            if verbose:
+                print(email, mnemonic, num, instructor)
+
+            # Use account if found. Otherwise, use dummy user
+            account_query = User.objects.filter(email=email)
+            if account_query.count() > 0:
+                if verbose:
+                    print('Account already found for', email)
+                account = account_query.first()
+            else:
+                account = dummy_account.first()
+
+            subdepartment_query = Subdepartment.objects.filter(mnemonic__iexact=mnemonic)
```

Comment:
I think stuff like this (and course_query, and account_query) should prob be inlined, and have a comment (if necessary):

```python
# account found
if User.objects.filter(email=email).count() > 0:
    ...
```

just a preference for helping these scripts be a more readable, they get super clunky